EXT_BeamPhysics.md

```diff
@@ -0,0 +1,269 @@
+  particles are in the same lattice element. Also see: `latticeElementID` and `locaitonInElement`.
+
+- `latticeElementID`
+  - Type Optional *(string)*
+  - Description: The ID string for the lattice element given by `latticeElementName`. The idea is that while more than
+    one lattice element may have the same name, the ID string will be unique.
+  - This, along with `locationInElement` sets the origin for specifying particle positions in the **lattice** coordinate system.
+  - Example: With [Bmad](https://www.classe.cornell.edu/bmad/) based programs the ID string is of the form
+    **branch-index>>element-index** where **branch-index** is the associated
+branch index integer, and **element-index** is the associated lattice element index within the branch.
+
+- `locationInElement`
+  - Type Optional *(int)*
+  - Description: This attribute is used with  `latticeElementID`/`latticeElementName` to specify
+  the origin where the particle or particles are measured with respect in the **lattice** coordinate system.   
+  - The origin is always on the longitudinal axis (x = y = 0) of the **lattice** coordinate system.
+  - Possible values:    
+    - `Upstream-End`: Upstream end of element outside of any edge fields.
+    - `<s-position>`: Where `<s-position>` is a number. Inside the element at a distance, given by `<s-position>`,
+    from the upsteam end of the element.
+    - `Downstream-End`: Downstream end of the element outside of any edge fields.
+  - Note: Since some programs will model edge fields of a lattice element as having zero length, the longitudinal **s**-position
+does not necessarily provide enough information to determine where a particle is with respect to an edge field so `Upstream-End` and `Downstream-End` are provided.
+
+- `momentumNormalization`
+    - Type: Optional *(string)*
+    - Description: Normalization used for momentum values.
+    - Possible values:
+        - `referenceTotalMomentum`: Normalize with respect to the `referenceTotalMomentum` attribute
+        - `referenceEnergy`: Normalize with respect to the `referenceEenergy` attribute.
+        - `none`: No normalization.
+
+- `referenceEnergy`
+    - Type: Optional *(float)* attribute.
+    - Description: Specifies the reference energy from which the `energy` is measured with respect to.
+
+- `referenceTotalMomentum`
+    - Type: Optional *(float)* attribute
+    - Description: Specifies the reference total momentum from which the total momentum is measured with respect to.
+
+- `totalCharge`
+    - Type: Optional *(float)* attribute.
+    - Description: The total charge of all the particles.
+
+Per-Particle Records of the `Particle Root Group`
+---------------------
+
+The following records store data on a particle by particle basis.
+
+- `time-refTime`
+- Type: Optional *(float)*
+- Description: Particle time minus the referece time.
+
+- `energy/`
```

Comment:
There is probably no need for the trailing `/` here. Is that correct?

Reply:
@ax3l @RemiLehe 

I see I was not consistent about this. I thought that using a trailing slash was the convention for records. It does not matter to me.  Whatever the convention is should be used.

Reply:
@ax3l Do we actually have any convention on this?

Reply:
@RemiLehe @ax3l: I checked, the base standard is also not consistent on this.

Reply:
Oh, I am fine with unifying this.

Reply:
The extension now uses trailing slash everywhere.